### ISSUE TITLE
Add feature: Deal with multiple plates in the same image

### ISSUE THREAD

**Issue by manupap1** [manupap1/libzoneminder-plugin-openalpr#6]
I coded the plugin with the possibility to deal with multiple plates detected in the same image but I didn't have the opportunity to test this configuration.
OpenALPR returns an array of detected plates, the main challenge is to keep the same relationship indexes/plates for all images in one event.

**Comment by source-code-sa** [comment]
Hi, What do you mean by "main challenge is to keep the same relationship indexes/plates for all images in one event". If related to insert into event database then the right perl should handle it. I recently built your plugin into source. Everything compiled fine and ZM started up with OpenALPR plugin loaded however for some reason my ZM is not recording events. I can force alarm to create event but modetect, mocord etc are not working. I will try to scratch off that VM and start over but in the meanwhile if you can explain above statement then i can perhaps help/contribute some of my expertise.

**Comment by knight-of-ni** [comment]
The ZoneMinder team has recently been informed that @manupap1 has passed away. 
He will be missed.

We have begun efforts to bring @nextime 's plugin support into ZoneMinder, and perhaps we could integrate @manupap1 's license plate plugin into the project following that.

**Comment by whopperg** [comment]
oh man, thats so sad. i just checked his git repo and read this...

**Comment by knight-of-ni** [comment]
@whopperg Yeah, it was very sad. If you are interested in continuing his work, please let us know. We are in a constant state of needing more help.

**Comment by whopperg** [comment]
Well I would if I had the knowledge. But I was just testing his plug in implementation, so I don't really have any idea what was going on with the coding site of things...
